content/practices/team-shared-history.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2018-06-28T00:00:00-05:00
+title: Team Shared History
+draft: false
+stages: ['delivery']
+---
+
+## What is it?
+
+A method of helping a team start to become cohesive, develop a shared sense of history, and a shared vision of the future. This simple exercise starts with a facilitator organizing the participants in order of their hire date with the company. Then a timeline is drawn and each team member marks their hire date on the timeline. Next, team members note on the timeline interesting things which may have occurred near their time of hire. Finally, the team member who has the most seniority will "tell the story" of the team's history and perhaps embellish with additional context around what has been going on around the organization.
+
+Outcomes
+- A better understanding between team members of their own place in the organization
+- Establish empathy and context for future conversations between team members
+- Build a sense of community for newer team members
+
+## Why Use Team Shared History?
+
+- New team members are not sure of their place, responsibilities, rights, and history of their org.
+- Team members with more history may assume that newer members know what is expected
+
+## Related Practices
+
+
+## Who do you need?
+
+- Facilitator
+- Product Owner
+- Project Sponsor, both technical and business
+- Architects
+- Developers
+- Scrum Master
+
+## Suggested Time
```

Comment:
an HOUR!? wow that is a long time. Does it really take that long? Exercises to get to know people are typically short.

Reply:
@mtakane This exercise is not intended as an ice-breaker. For teams that have little/no history together, it would be used as a level-setting discussion to better understand each other. The way that I saw it described and facilitated was:

* Facilitator draws a timeline (butcher paper would be good here, but a whiteboard can work)
* Facilitator hands out stickies and markers/pens
* Each team member writes up to 5 events during their time with the organization which they feel are "formative" or "indicative" about the organization
* Team members work together to put the events on the timeline
* After the timeline is completed, the "senior" team member works with their teammates to recount the events in order to form a more cohesive narrative
* The timeline is photographed and/or archived for later updates with new team members